### ISSUE TITLE
code-code

### ISSUE THREAD

**Issue by lizhh268** [ntunlp/xCodeEval#13]
Hello, I am very interested in the datasets you contributed, but I have a question regarding the code-to-code retrieval task. Specifically, the validation set has positive and negative sample pairs, while the training and test sets do not have positive and negative sample pairings.
